apps/web/pages/api/auth/[...nextauth].tsx

```diff
@@ -140,7 +140,7 @@ if (isSAMLLoginEnabled) {
     },
   });
 }
-
+const cookiePrefix = "";
```

Comment:
Medium: I'm thinking this is a global constant meant to set by the server configurator? If that is the case (and otherwise it is confusing me so probably still needs looking at) maybe add this as a constant in the `from "@server/lib/constants";` file? `COOKIE_PREFIX`?